tests/baselines/reference/privateNamesConstructorChain-1.errors.txt

```diff
@@ -0,0 +1,20 @@
+tests/cases/conformance/classes/members/privateNames/privateNamesConstructorChain-1.ts(3,5): error TS18019: 'static' modifier cannot be used with a private-named property.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNamesConstructorChain-1.ts (1 errors) ====
+    class Parent {
+        #foo = 3;
+        static #bar = 5;
+        ~~~~~~
+!!! error TS18019: 'static' modifier cannot be used with a private-named property.
+        accessChildProps() {
+            new Child().#foo; // OK (`#foo` was added when `Parent`'s constructor was called on `child`)
+            Child.#bar;       // Error: not found

```

Comment:
Should this also be an error?

Reply:
Fixed, but I took a bit of a shortcut because this PR bans static private identifier fields. Made a note about it in the privateNamesAndStaticFields test.

Reply:
Hi @rbuckton , thanks for all your help. Would you be up for taking another look? We've incorporated the last round of feedback.

The only thing we are still working on is a minor [tooltip issue that affects the playground](https://github.com/microsoft/TypeScript/pull/30829#issuecomment-534157560) and we hope to have that fixed tomorrow.